Lib/test/test_pdb.py

```diff
@@ -4275,9 +4284,18 @@ def _create_fake_frozen_module():
 
         self.addCleanup(os_helper.unlink, 'gh93696.py')
         self.addCleanup(os_helper.unlink, 'gh93696_host.py')
-        stdout, stderr = self._run_pdb(["gh93696_host.py"], commands)
-        # verify that pdb found the source of the "frozen" function
+
+        # verify that pdb found the source of the "frozen" function and it
+        # shows the breakpoint at the correct line for both list and longlist
+        stdout, _ = self._run_pdb(["gh93696_host.py"], commands_list)
+        self.assertIn('x = "Sentinel string for gh-93696"', stdout, "Sentinel statement not found")
+        self.assertIn('4 B', stdout, "breakpoint not found")
+        self.assertIn('-> def func():', stdout, "stack entry not found")
+
+        stdout, _ = self._run_pdb(["gh93696_host.py"], commands_longlist)
         self.assertIn('x = "Sentinel string for gh-93696"', stdout, "Sentinel statement not found")
+        self.assertIn('4 B', stdout, "breakpoint not found")
+        self.assertIn('-> def func():', stdout, "stack entry not found")
```

Comment:
These two tests look identical as l. Make a loop?

Reply:
Good point! I used loop for it.